cmip6_downscaling/data/cmip.py

```diff
@@ -99,95 +125,52 @@ def load_cmip(
         return ds_out
 
 
-def convert_to_360(lon: Union[float, int]) -> Union[float, int]:
-    """Convert lons to 0-360 basis.
-    Parameters
-    ----------
-    lon : float or int
-        Longitude on -180 to 180 basis
-    Returns
-    -------
-    lon : float or int
-        Longitude on 0 to 360 basis
-    """
-    if lon > 0:
-        return lon
-    elif lon < 0:
-        return 360 + lon
-
-
-def gcm_munge(ds: xr.Dataset) -> xr.Dataset:
-    """Clean up GCM dataset by swapping lats if necessary to match ERA5 and
-    deleting unnecessary variables (e.g. height).
-    Parameters
-    ----------
-    ds : xr.Dataset
-        GCM dataset direct from catalog (though perhaps subsetted temporally)
-    Returns
-    -------
-    ds : xr.Dataset
-        Super clean GCM dataset
-    """
-    # TODO: check if we need to flip this to > now that we have a preprocessed version of ERA5
-    # TODO: for other gcm grids check the lons
-    if ds.lat[0] < ds.lat[-1]:
-        ds = ds.reindex({"lat": ds.lat[::-1]})
-    ds = maybe_drop_band_vars(ds)
-    if 'height' in ds:
-        ds = ds.drop('height')
-    ds = ds.squeeze(drop=True)
-    return ds
-
-
 def get_gcm(
```

Comment:
Should these funcs be passed the run_parameters object?

Reply:
I'd like to keep the run_parameters object inside the task/flow scope.